navigation/navigation-compose/api/desktop/navigation-compose.api

```diff
@@ -97,8 +101,11 @@ public final class androidx/navigation/compose/NavGraphBuilderKt {
 	public static synthetic fun navigation$default (Landroidx/navigation/NavGraphBuilder;Lkotlin/reflect/KClass;Lkotlin/reflect/KClass;Ljava/util/Map;Ljava/util/List;Lkotlin/jvm/functions/Function1;Lkotlin/jvm/functions/Function1;Lkotlin/jvm/functions/Function1;Lkotlin/jvm/functions/Function1;Lkotlin/jvm/functions/Function1;Lkotlin/jvm/functions/Function1;ILjava/lang/Object;)V
 }
 
-public final class androidx/navigation/compose/NavHostController_jbKt {
+public final class androidx/navigation/compose/NavHostControllerKt {
 	public static final fun currentBackStackEntryAsState (Landroidx/navigation/NavController;Landroidx/compose/runtime/Composer;I)Landroidx/compose/runtime/State;
+}
+
+public final class androidx/navigation/compose/NavHostController_jbKt {
```

Comment:
It seems we need to add
```
@file:JvmName("NavHostControllerKt")
@file:JvmMultifileClass
```

to jbMain too to preserve binary compatibility later

Reply:
👍